lib/util/keywords.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
```

Comment:
Sorry about that. Should I attribute the file to myself? The list is just copy&paste from the original list in `dot-notation`.